### ISSUE TITLE
Can't find Homebuilder and Rule Engine

### ISSUE THREAD

**Issue by DominikMie** [openhab/openhab-syno-spk#122]
In 2.2.0.008 SNAPSHOT both could be found in the Addons. In 2.2.0.010 Release both packages are gone. 

Can anybody tell how to find them?

**Comment by cniweb** [comment]
@kaikreuzer Is this a problem of this package or of openHAB 2.2?

**Comment by kaikreuzer** [comment]
I'd say of this package. Both exist in the 2.2.0 release distro.

**Comment by kaikreuzer** [comment]
Let me correct myself: I'd say it is neither the package nor the release.
I have just installed 2.2.0.010 on my Synology and I have Home Builder as well as the rule engine available and installed without any problem.

I would hence claim that something is screwed up on @DominikMie's installation.

**Comment by DominikMie** [comment]
Mea culpa! Mea maxima culpa!

I ugraded from 2.2.0.008 SNAPSHOT and had to rename the existig directories. Furthermore I copied the content of old .../openHAB2/userdata/ to new .../openHAB/userdata/. As a result OH 2.2.0.10 Release ran with all existing bindings, things and items. But it was not possible to install new bindings. 

So I made a fresh installation and had to re-define all  things and items after re-installing all bindings. GRMPF!!! But now everything works fine. 

Thx for care.
